theme/static/css/hydroshare_core.css

```diff
@@ -4345,4 +4345,10 @@ div#fb-file-operations-controls {
 
 #rename-dialog .input-group {
 	width: 100%;
+}
+
```

Comment:
@mauriel133 I do not think we want to resolve the map loading issue for this PR. I use different IDs for the loading spinners to control each of them, and use the same class for dealing with CSS for them. And I changed the map legend text styles. Can you review it now? Thanks!